src/components/RadioButton/RadioButton.ts

```diff
@@ -9,30 +9,59 @@ namespace fabric {
    * Adds basic demonstration functionality to .ms-RadioButton components.
    *
    */
-  export class RadioButton extends CheckBox {
+  export class RadioButton {
```

Comment:
I see! Seems like we will have to have a base class, lets say ChoiceField, without any style from which both CheckBox and RadioButton could be extended. Keeping common functionality in base class, but style and other functionality in child classes.